index.html

```diff
@@ -191,7 +121,7 @@ <h2 class="about-me-topper">About me</h2>
             </div>
 
             <div class="card framework-card">
-                <img src="images/frameworks.png" alt="framework_icon">
+                <img src="images/frameworks.png" alt="framework_icon" />
                 <div class="card-details">
                     <p>Frameworks</p>
```

Comment:
- [ ] Great job working on this milestone. In the mobile view, your `See Live` and `See Source` buttons are aligned to the left, but in the [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=ZgI21TLlpHaIHveH-0) design they are justified. Kindly fix this issue.
- Figma design
![image](https://user-images.githubusercontent.com/62849872/224324859-dac5b173-cf6e-4119-9240-2ecea0465ea3.png)

- Your design
![image](https://user-images.githubusercontent.com/62849872/224325022-6967656b-f983-4e23-91c9-cae3958fb025.png)